.eslintrc.cjs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: '@studiometa/eslint-config',
+  rules: {
+    'import/extensions': ['error', 'always', { ignorePackages: true }],
+    'no-underscore-dangle': 'off',
+  },
+  overrides: [
+    {
+      files: [ '**/*.spec.js', '**/spec.js', 'packages/tests/**/*.js' ],
+      extends: [ 'plugin:jest/recommended', 'plugin:jest/style' ],
+      rules: {
+        'max-classes-per-file': 'off',
+        'jest/no-test-callback': 'off',
+        'require-jsdoc': 'off',
```

Comment:
Is there a reason to set this off ?

Reply:
We do not need JS doc blocks in the test files 😉